playbooks/installation/index.adoc

```diff
@@ -454,6 +448,13 @@ openshift-appnode-1.c1-ocp.myorg.com.com
 openshift-appnode-2.c1-ocp.myorg.com
 ----
 
+Test ansible in an adhoc way to ensure it can get to all the nodes
+
+----
+ansible -i /some/path/to/my/hosts/inventory/file OSEv3 -m ping
```

Comment:
we've established a hosts file path and name. can we sue that instead of `/some/path/to/my/hosts/inventory/file `?